packages/core/src/routes/applications/types.ts

```diff
@@ -37,6 +37,28 @@ const applicationCreateGuardWithProtectedAppMetadata = originalApplicationCreate
       .optional(),
   });
 
+const applicationPatchGuardWithProtectedAppMetadata = originalApplicationPatchGuard
+  .deepPartial()
+  .omit({
+    protectedAppMetadata: true,
+  })
+  .extend({
+    protectedAppMetadata: z
+      .object({
+        origin: z.string().optional(),
```

Comment:
Move the deepPartial statement to the last one?

Reply:
The array item of `pageRules` can not be optional, so `deepPartial` should be in the first.